src/components/VIcon/VIcon.spec.js

```diff
@@ -25,10 +25,16 @@ test('VIcon.js', ({ mount, compileToFunctions }) => {
     expect(wrapper.element.classList).toContain('icon--disabled')
   })
 
+  it('should render no size by default', () => {
+    const context = functionalContext({}, 'add')
+    const wrapper = mount(VIcon, context)
+
+    expect(wrapper.element.style.fontSize).toBe('')
```

Comment:
should not set font size if none provided

Reply:
I've updated the spec name